src/AlterableTranslatorInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Drupal\oe_translation;
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Routing\RouteMatchInterface;
+
+/**
+ * Interface for TMGMT translator plugins that can alter translation pages.
+ */
+interface AlterableTranslatorInterface {
```

Comment:
Both the LocalTranslatorInterface and AlterableTranslatorInterface define methods that alter forms and entities associated with the Translator. Is there any particular need to have two separate interfaces?

Reply:
@imanoleguskiza Yes. One is for translators that work locally only which has methods remote translators won't have. And the other is for any translators.